lib/main.dart

```diff
@@ -1,9 +1,7 @@
+import 'package:climbing_diary/pages/diary_page.dart';
 import 'package:climbing_diary/pages/map_page.dart';
 import 'package:climbing_diary/pages/statistic_page.dart';
 import 'package:flutter/material.dart';
-import 'package:flutter_map/flutter_map.dart'; // Suitable for most situations
```

Comment:
Funktioniert es auch ohne die Bibliotheken? @FlorianPix

Reply:
ja die Bibliothek wird nur in MapPage gebraucht